app/models/submission.rb

```diff
@@ -23,6 +23,10 @@ class Submission < ApplicationRecord
   validates :sunetid, presence: true
   validates :title, presence: true
 
+  def to_param
```

Comment:
A comment here to show this removes the need for `@submission.dissertation_id` in the helpers would be good, but not a blocker.

Reply:
added it, thx!